packages/core/src/components/templates/SearchPage/SearchWrapper.tsx

```diff
@@ -45,11 +48,17 @@ export default function SearchWrapper({
       selectedFacets,
     }
   )
-
+  
   if (isValidating || !pageProductGalleryData) {
     return <EmptySearch />
   }
 
+  const hasFuzzy = findFacetValue(selectedFacets, 'fuzzy')
+
+  if(!hasFuzzy) {
+    return <EmptySearch />
+  }
+
```

Comment:
The only change that is different from https://github.com/vtex/faststore/pull/2204